tagr/tagging/artifacts.py

```diff
@@ -105,6 +105,9 @@ def flush(self, proj, exp, dump='local', tag=None):
         if not tag:
             logger.info("using datetime as tag")
             tag = str(datetime.utcnow())
+        
+        # if folder directory doesnt exist, then create new directory
+        self.storage_provider.build_path(proj, experiment, tag)
```

Comment:
Does this need to run if `storage_provider` is not `local`? I dont think we should create a local dir if we dont have to (in the case where we just push to s3)

Reply:
yep, just realized that, lets only run it if ```dump == local```